tests/application_development/code_relocation/linker_riscv_qemu_sram2.ld

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) ??? ???
```

Comment:
?????? :)

Reply:
That's what I get for deferring figuring out what I needed to fill in there! Updated to have an actual value.